examples/plot_hidden_snakes.py

```diff
@@ -0,0 +1,321 @@
+"""
+==============================================
+Conditional Interactions on the Snakes Dataset
+==============================================
+
+This is a variant of plot_snakes.py 
+
+Snake are hiding!! Therefore, some picture have colored pixels despite they do not contain any snake.
+
+    JL Meunier - January 2017
+    
+    Developed for the EU project READ. The READ project has received funding 
+    from the European Union's Horizon 2020 research and innovation programme 
+    under grant agreement No 674943
+    
+    Copyright Xerox
+
+This example uses the snake dataset introduced in
```

Comment:
wow I think they didn't expect this snake to live on so long lol.

Reply:
:))